py/generate.py

```diff
@@ -1022,6 +1022,13 @@ def main(browser_protocol_path, js_protocol_path, output_path):
                 if event.name == 'screencastVisibilityChanged':
                     # Patch 2
                     event.description = event.description.replace('`', '')
+        elif domain.domain == 'Debugger':
+            for event in domain.events:
+                for parameter in event.parameters:
+                    if parameter.name in ("scriptLanguage", "debugSymbols"):
+                        # Patch 3 required for debugging referencing itself internally.
+                        if parameter.ref.startswith("Debugger."):
```

Comment:
I've added this fix for when the generator gets fixed upstream / rolled up to the newer versions; I will look to provide a patch back to it and roll it's devtools version up too next week.